airflow/providers/fab/__init__.py

```diff
@@ -32,8 +32,8 @@
 __version__ = "1.3.0"
 
 if packaging.version.parse(packaging.version.parse(airflow_version).base_version) < packaging.version.parse(
-    "2.9.0"
+    "3.0.0"
```

Comment:
Hm. That would mean that we won't have FAB provider released for Airflow 2 and I think it's not good, because for example new FAB might get released with critical security fixes and we won't be able to upgrade it

I think we should find a solution with compatibility layer for Airlfow 2 - either via some conditional imports in FAB or implementing some `airflow.providers.compat` schim.

Reply:
Sure. I was trying to see if the compact test failures would be fixed. I will remove it and implement proper compatibility shim